manifests/profiles/default.yaml

```diff
@@ -232,9 +232,9 @@ spec:
       validationURL: ""
     pilot:
       autoscaleEnabled: true
-      autoscaleMin: 1
-      autoscaleMax: 5
-      replicaCount: 1
+      autoscaleMin: 2
```

Comment:
`autoscaleMin: 1` is not sufficient to drain a node? Does K8s generate a validation error?

Reply:
Well if it scaled back down to 1, you'd have only 1 replica and therein again run into the PDB issue